kujira/assetlist.json

```diff
@@ -2362,6 +2362,49 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nami.svg"
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+          "exponent": 0
+        },
+        {
+          "denom": "YUM.axl",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/507BE7E33F06026652F519AD4D36716251F2D34DF04514A905D3B19A7D8130F7",
+      "name": "Axelar Wrapped YUM",
+      "display": "yum",
+      "symbol": "YUM.axl",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "axelar",
+            "base_denom": "yum-wei",
+            "channel_id": "channel-14"
+          },
+          "chain": {
+            "channel_id": "channel-9",
+            "path": "transfer/channel-9/yum-wei"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "ethereum",
+            "base_denom": "0xcE682c89C63d2850Cb2ca898E44D6c7c30d897a6"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yum.png"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yum.png"
```

Comment:
This image has been upload elsewhere, so we no longer need the image file you're adding in this PR